components/_previews/_inputs-validation-format/_inputs-validation-format.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="dso-example-wrapper">
+  <div class="dso-example-text">
+    <h2>Inline validatie</h2>
+  </div>
+  <div class="dso-example">
+    <table class="table">
+      <caption>Formulierelementen met specifiek gewenst formaat.</caption>
+      <thead>
+        <tr>
+          <th scope="col">Beschrijving</th>
+          <th scope="col">Weergave</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Veld met fout</td>
+          <td>
+            {{> '@group-input' label='E-mailadres' value="piet@" state='invalid' errorText='Er is geen geldig e-mailadres ingevuld. Controleer of het e-mailadres een @ en een punt bevat, en of er geen spaties in staan.'}}
```

Comment:
Net ook face-to-face besproken, maar ik denk dat het beter is om het complete formulier op te bouwen. Styling en dergelijke komt dat zeker goed

Reply:
done